ui/src/main/resources/CKEditor/Translations.xml

```diff
@@ -58,7 +58,7 @@ ckeditor.admin.reset.confirmation=Are you sure you want to restore the default c
 ckeditor.plugin.link.options=Options
 ckeditor.plugin.link.anchor=Anchor
 ckeditor.plugin.link.queryString=Query String
-ckeditor.plugin.link.defaultLabel=type the link label
+ckeditor.plugin.link.defaultLabel=Type the link label
```

Comment:
https://snag.gy/ioyfVq.jpg See?  "Find a page" with a uppercase and "type the link label" with lowercase but I understand what you mean ... You can reject my PR if you wish :-)